docs/src/modules/developing/pages/development-process-js.adoc

```diff
@@ -79,6 +78,6 @@ xref:javascript:run-locally.adoc[running them locally] before deploying to Akka
 
 After testing locally, deploy your service to Akka Serverless using the CLI or the Console. The following pages provide information about deployment:
 
-* https://developer.lightbend.com/docs/akka-serverless/getting-started/projects.html[Working with Akka Serverless deployment projects]
-* https://developer.lightbend.com/docs/akka-serverless/deploying/deploying.html[Deploying a packaged service]
-* https://developer.lightbend.com/docs/akka-serverless/tutorial/index.html[Examples of how to deploy]
+* https://developer.lightbend.com/docs/akka-serverless/projects/index.html[Working with Akka Serverless deployment projects]
```

Comment:
@retgits subjective decision here, could you please confirm?

Reply:
I'd choose this one too, also could you call it `Working with Akka Serverless projects` 😇